tools/run-e2e-test.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run the e2e test by building the e2e.test binary, then starting the
+# e2e test from the command line using "kubetest2" using the "gce"
+# deployer. NOTE: You must specify a GCP project (and zone) that
+# you've already logged in to to bring up the cluster. The cluster
+# will use k8s binaries built and stored at the latest stable version.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Prerequisites
+#
+# This script should be run in the repo root directory:
+#
+# $ ./tools/run-e2e-test.sh
+#
+# APPLICATION_DEFAULT_CREDENTIALS must be set for the ginkgo tester
+# to be able to run against the created cluster. These credentials
+# are created with the following command and login flow:
+#
+#     $ gcloud auth application-default login
+#
+# The credentials are usually stored in the ~/.config/gcloud
+# directory in a file called:
+#     application_default_credentials.json
+# The credentials can also be set as a value of the
+# APPLICATION_DEFAULT_CREDENTIALS environment variable.
+
+# Parameters
+#
+# The cluster will run the latest stable released k8s binaries.
+VERSION=$(curl -L -s https://dl.k8s.io/release/stable.txt)
+VERBOSITY=6
+GCP_PROJECT=seans-devel
```

Comment:
we may want to use boskos here https://github.com/kubernetes/cloud-provider-gcp/blob/master/test/boskos.sh

Reply:
Got it. I wanted to first get the e2e tests running manually (which currently works), then get it working with a boskos project. I'll now get work on getting it to work using a boskos project.